stock_request_picking_type/models/stock_request_order.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2019 Open Source Integrators
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+from odoo import api, fields, models
+
+
+class StockRequestOrder(models.Model):
+    _inherit = "stock.request.order"
+
+    @api.model
+    def _get_default_picking_type(self):
+        companies = self._context.get("allowed_company_ids", []).copy()
```

Comment:
_context is deprecated since a long time: https://github.com/odoo/odoo/blob/16.0/odoo/models.py#L5160

Reply:
Still using here:
https://github.com/odoo/odoo/blob/17.0/odoo/models.py#L1158

Reply:
That's not why they are using it, we should do it too :see_no_evil:

Reply:
With `self.env.context` much better no?